test/tests/outputs/csv/tests

```diff
@@ -105,4 +105,12 @@
     requirement = "The system shall support CSV output to files on nonlinear iterations, generating unique filenames containing the time step and current nonlinear iteration number."
     issues = "#24705"
   []
+  [pp_scalar_execute_on_mismatch]
+    type = RunException
+    input = 'csv.i'
+    cli_args = 'Outputs/csv/execute_postprocessors_on=FINAL'
+    expect_err = "The parameters 'execute_postprocessors_on', 'execute_scalars_on', and 'execute_reporters_on' must be the same for CSV output"
+    requirement = "The system shall report an error if post-processors and scalar variables have different output frequency."
+    issues = "#25211"
+  []
```

Comment:
I know it's not great.. but aren't there multiple combos to test here?

Reply:
Yes, I'd like to choose only the most important combos. Someone may address the issue in a more elegant manner (probably not any time soon though...). What tests would you like?

Reply:
Let's just add one for reporters and that's good enough